infra/config/validation.yaml

```
@@ -3,70 +3,82 @@
     scopes: [ ]
-    example: "revert: 'fix(Database): Resolve connection timeout issue'"
+    examples:
+      - "revert: 'fix(Database): Resolve connection timeout issue'"
+
   - name: remove
     description: Remove deprecated or unnecessary code, files, or features.
     requires_scope: true
     scopes: [ Code, Files, Feature ]
-    example: "remove(Code): Delete obsolete authentication module"
+    examples:
+      - "remove(Code): Delete obsolete authentication module"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)